index.html

```diff
@@ -186,11 +186,14 @@ <h2 id="constraints">Constraining Display Surface Selection</h2>
 
       <section>
         <h2>New Constraints for Captured Display Surfaces</h2>
-        <p>Two constraints are defined that allow an application to observe
+        <p>Three constraints are defined that allow an application to observe
         properties of the selected <a>display surface</a>.  Since the source of
         media cannot be changed after a <code>MediaStreamTrack</code> has been
         returned, these constraints cannot be changed by an application.</p>
 
+        <p>The <code><a href="#widl-MediaTrackConstraintSet-disableLocalEcho">disableLocalEcho</a></code>
+        constraint allows an application to suppress local playback of the captured audio.</p>
```

Comment:
A name of "muteAudioForSource" might be easier to understand. Having booleans with "disable" in the name is usually bad practice.